cmd/glasskube/cmd/bootstrap.go

```diff
@@ -31,6 +33,19 @@ var bootstrapCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
 		client := bootstrap.NewBootstrapClient(cfg)
+		installedVersion, err := clientutils.GetPackageOperatorVersion(cmd.Context())
+		if err != nil {
+			fmt.Printf("error : %v\n", err)
+			return
+		}
+		desiredVersion := config.Version
```

Comment:
`config.Version` is only the desired version, if the given url is not set. So please check that `bootstrapCmdOptions.url` is empty before doing the version check. If the user sets a custom url, we cannot parse what version they are about to install.

Reply:
Hey can you explain this, what I have done far now is:
```
var desiredVersion string
		if bootstrapCmdOptions.url == "" {
			desiredVersion = config.Version
		} else {

		}
```

Reply:
@hanshal101 please commit your current status to the branch so I can see the whole context.